File: src/main/java/com/dennisturco/dto/ProductResponseDTO.java
package com.dennisturco.dto;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class ProductResponseDTO {
    private Long id;
    private Long categoryId;
    private String categoryName;
    private Long statusId;
    private String statusName;
    private String code;
    private String ean;
    private String name;
    private String description;
    private Integer quantity;
    private Integer vatPercentage;
    private Float price;
}

File: GestioPro.Common/DTOs/ProductResponseDTO.cs
using GestioPro.Common.Enums;

namespace GestioPro.Common.DTOs;

public record ProductResponseDTO(
    long Id,
    long CategoryId,
    string CategoryName,
    ProductStatus ProductStatus,
    ItemType ItemType,
    string Code,
    string? Ean,
    string Name,
    string? Description,
    int? Quantity,
    int VatPercentage,
    float Price,
    bool IsDisabled
);
